docs/_docs/2_4_configurations.md

```diff
@@ -510,3 +510,28 @@ Property: `hoodie.memory.compaction.fraction` <br/>
 #### withWriteStatusFailureFraction(failureFraction = 0.1) {#withWriteStatusFailureFraction}
 Property: `hoodie.memory.writestatus.failure.fraction` <br/>
 <span style="color:grey">This property controls what fraction of the failed record, exceptions we report back to driver</span>
+
+#### Write commit callback configs
+Controls callback behavior on write commit
```

Comment:
would describe in details, such as if enables the callback but failed to callback.

Reply:
> would describe in details, such as if enables the callback but failed to callback.

done